Corrections.py

```diff
@@ -153,10 +153,10 @@ def applyScaleUncertainties(self, df, ana_reco_objects):
         source_dict = { central : [] }
         if 'tauES' in self.to_apply:
             df, source_dict = self.tau.getES(df, source_dict)
-        if 'JEC_JER' in self.to_apply:
-            df, source_dict = self.jet.getP4Variations(df, source_dict)
-            df, source_dict = self.fatjet.getP4Variations(df, source_dict)
-        if 'tauES' in self.to_apply or 'JEC_JER' in self.to_apply:
+        if 'JEC' in self.to_apply:
```

Comment:
what if we want to apply JER only without JEC?

Reply:
... ok I need to add also this option